### PR TITLE
#149975506 Create notification dropdown

### PR DESCRIPTION
**Pull request by andela-cofor**
#### What does this PR do?
- add a drop down list to the notification icon. 

#### Description of Task to be completed?
create a drop down feature to display a drop down list when user hovers on notification icon on the navigation bar.

#### How should this be manually tested?
After cloning the repository and installing the required dependencies using 
* Run `yarn install`
* Run  `npm run start:dev -s`
* Hover on the notification icon

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
 [#149975506 ](https://www.pivotaltracker.com/story/show/149975506) 

#### Screenshots (if appropriate)
screen shot 2017-08-11 at 11 23 29 am

#### Questions
N/A

### CODE REVIEW

File: src/common/navbar/NavBar.scss [hunk 1]
```diff
@@ -174,6 +174,96 @@
         }
       }
 
+      .notifications {
+        right: -300%;
+        width: 27rem;
+
+        .content-container {
+          padding: 0rem;
```
Comment (Mbarak-Mbigo) [inline src/common/navbar/NavBar.scss]: For 0 values, you don't need to specify a unit of measurement. `padding: 0;` is just fine.

File: src/common/navbar/NavBar.scss [hunk 2]
```diff
@@ -174,6 +174,96 @@
...
+            cursor: pointer;
+
+            img {
+              height: 2rem;
+              border-radius: 50%;
+              float: left;
+            }
+
+            h3 {
+              margin-top: -0rem;
```
Comment (Mbarak-Mbigo) [inline src/common/navbar/NavBar.scss]: What's the purpose of a `negative 0`? `margin-top: -0rem;`

**Comment by andela-cofor** [comment]
@Mbarak-Mbigo done

File: src/common/navbar/NavBar.scss [hunk 3]
```diff
@@ -146,10 +146,9 @@
 
       .content-container {
         background-color: map-get($grey, 100);
+        border: solid 1px map-get($grey, 200);
         position: relative;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: This shouldn't be here.
Following the guidelines defined [here](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/): positioning should come before display & box-model

File: src/common/navbar/NavBar.scss [hunk 4]
```diff
@@ -146,10 +146,9 @@
...
         z-index: 10;
-        padding: 0.7rem;
-        border: solid 1px map-get($grey, 200);
         border-radius: 3px;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: Wouldn't you like to represent this value in rem? `0.1875rem;`

File: src/common/navbar/NavBar.scss [hunk 5]
```diff
@@ -169,16 +168,113 @@
...
+          border-bottom: 1px solid map-get($grey, 200);
+          padding-left: 2rem;
+
+          div {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            height: 2.5rem;
+            width: 95%;
+            font-family: Primary-DINPro, sans-serif;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: If you import the `theme.css` file, there won't be a need for this here.

File: src/common/navbar/NavBar.scss [hunk 6]
```diff
@@ -169,16 +168,113 @@
...
+          }
+
+          .mark-as-read {
+            color: map_get($colors, danger);
+            cursor: pointer;
+          }
+        }
+
+        .notification-body {
+          height: 19em;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: Make use of `rem` instead of `em`.

File: src/common/navbar/NavBar.scss [hunk 7]
```diff
@@ -169,16 +168,113 @@
...
+          padding-top: 1rem;
+          padding-left: 1rem;
+          letter-spacing: 0.9px;
+
+          .content {
+            display: flex;
+            height: 5.5rem;
+            width: 91%;
+            margin-bottom: 1.5rem;
+            color: #9f9f9f;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: colors can be gotten from `theme.css`

File: src/common/navbar/NavBar.scss [hunk 8]
```diff
@@ -169,16 +168,113 @@
...
+            cursor: pointer;
+
+            img {
+              height: 2rem;
+              border-radius: 50%;
+              float: left;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: ### Nitpick

The styling is not properly ordered. `float` shouldn't come last.

File: src/common/navbar/NavBar.scss [hunk 9]
```diff
@@ -169,16 +168,113 @@
...
+            }
+
+            h3 {
+              margin-top: 0rem;
+              padding-left: 1rem;
+              font-family: Primary-DINPro, sans-serif;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: Already inheriting from the `theme.css`

File: src/common/navbar/NavBar.scss [hunk 10]
```diff
@@ -169,16 +168,108 @@
...
+            color: map_get($colors, danger);
+            cursor: pointer;
+          }
+        }
+
+        .notification-body {
+          height: 20rem;
+          padding-top: 1rem;
+          padding-left: 1rem;
+          letter-spacing: 0.9px;
```
Comment (BolajiOlajide) [inline src/common/navbar/NavBar.scss]: `0.9px = 0.05625rem;`

**Comment by BolajiOlajide** [comment]
LGTM.

**Comment by andela-cofor** [comment]
@andela-kakpobome done

**Pull request merged**